src/checkout-latest-vscode.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Checks-out the vscode git submodule to the latest "solid" release commit/tag.
+ * e.g. 1.45.0, 1.46.4
+ */
+const { run, vscode } = require('./paths.js');
+
+// @ts-check
+try {
+    checkoutLatestVscodeRelease();
+} catch (e) {
+    console.log(`Error: ${e}`)
+    process.exit(1);
+}
+
+async function checkoutLatestVscodeRelease() {
+    try {
+        const latestTagSha = (await run('git', ['rev-list', '--tags', '--max-count=1'], vscode())).trim();
+        const latestTag = (await run('git', ['describe', '--tags', latestTagSha], vscode())).trim();
```

Comment:
[git rev-list --tags doc:](https://git-scm.com/docs/git-rev-list#Documentation/git-rev-list.txt---tagsltpatterngt)
"Pretend as if all the refs in refs/tags are listed on the command line as \<commit\>. If \<pattern\> is given, limit tags to ones matching given shell glob. If pattern lacks ?, \*, or [, /* at the end is implied."

When I look at the `refs/remotes` folder for the vscode git submodule, I see:

vscode-builtin-extensions/vscode$ l ../.git/modules/vscode/refs/tags/
1.41.0  1.41.1  1.42.0  1.42.1  1.43.0  1.43.1  1.43.2  1.44.0  1.44.1  1.44.2  1.45.0  1.45.1  HEAD  help

Good suggestion, since we can't be sure that some non-release tags might get pushed at some point on master, like the `help` one above.

Tested and confirmed the suggested regexp does the trick:
![image](https://user-images.githubusercontent.com/25749063/83646253-67db2200-a581-11ea-9968-ee4da3379e22.png)

Reply:
done